rc_django/views.py

```diff
@@ -78,9 +75,9 @@ def proxy(request, service, url):
 
     start = time()
     try:
-        if service == "iasystem" and subdomain is not None:
-            response = dao.getURL(url, headers, subdomain)
-
+        if re.match(r'^iasystem_', service):
```

Comment:
I think you should get rid of this block, and just set the headers for iasystem on line 51.